spec/live/topic_subscription_spec.rb

```diff
@@ -8,16 +8,4 @@
   end
 
   it_should_be_readable topic, :subscriptions
-
-  # TODO: This resource cannot be found since the response from the server
-  # is using `subscription` as the model name, instead of `topic_subscription`
-  # which is what the save handle_response method is expecting:
-  # https://github.com/zendesk/zendesk_api_client_rb/blob/master/lib/zendesk_api/actions.rb#L5
-  # we should modify it to use the `model_key` if present, but I don't want to do
-  # that now, when I modifying a lot of tests
-  xit "subscription can be found via the topic" do
-    VCR.use_cassette("find_created_topic_subscription") do
-      expect(topic.subscriptions.find(id: subscription.id)).not_to be nil
-    end
-  end
```

Comment:
I am a bit confused about why are we removing this.

Reply:
The x in `xit` stands for "temporarily skipped". This has been skipped for years now, so it's not temporary, it's dead code that doesn't work and just makes noise in the test report.

Reply:
If it is a legit todo today, maybe we can create an issue out of this. I'll leave it to you.